js/layout-javascript/small-card-code.js

```diff
@@ -2027,12 +2028,15 @@ DynamicList.prototype.showDetails = function(id, listData) {
 
         var template = Handlebars.compile(data.src || src);
         var wrapperTemplate = Handlebars.compile(wrapper);
+        var $pageContentBody = $('.fl-page-content-wrapper').parent();
```

Comment:
Why do we use `$('.fl-page-content-wrapper').parent()` and not `$('body')`?

Reply:
> Why do we use `$('.fl-page-content-wrapper').parent()` and not `$('body')`?

I use ```$('.fl-page-content-wrapper').parent()``` because ```$('body')``` is selected only for a specific list on the page, but not for all content in the iframe

Reply:
@skrupnyk I don't understand the scenario you are describing.

![image](https://user-images.githubusercontent.com/290733/128333575-835e4bcb-85b8-4e94-91c4-4bf8db09fe8a.png)

This is a snapshot of our engine template. As you can see, the container with `.fl-page-content-wrapper` is always the direct child of `<body>` if the app supports the latest D&D. If it doesn't, then there is no `.fl-page-content-wrapper`.

Using `$('.fl-page-content-wrapper').parent()` would either:

1. Always lead to `<body>` if the app supports the latest D&D; or
2. Lead to nothing if it doesn't support the latest D&D.

So why wouldn't we just reference the `body` element directly? What's the scenario you have in mind that means we should use `$('.fl-page-content-wrapper').parent()`?

Reply:
> @skrupnyk I don't understand the scenario you are describing.
> 
> ![image](https://user-images.githubusercontent.com/290733/128333575-835e4bcb-85b8-4e94-91c4-4bf8db09fe8a.png)
> 
> This is a snapshot of our engine template. As you can see, the container with `.fl-page-content-wrapper` is always the direct child of `<body>` if the app supports the latest D&D. If it doesn't, then there is no `.fl-page-content-wrapper`.
> 
> Using `$('.fl-page-content-wrapper').parent()` would either:
> 
> 1. Always lead to `<body>` if the app supports the latest D&D; or
> 2. Lead to nothing if it doesn't support the latest D&D.
> 
> So why wouldn't we just reference the `body` element directly? What's the scenario you have in mind that means we should use `$('.fl-page-content-wrapper').parent()`?

understood, make sense